README.md

```diff
@@ -1,6 +1,7 @@
-# Ubuntu Advantage Client
+<h1 align="center"><a href="https://ubuntu.com/advantage" target="_blank">Ubuntu Advantage Client</a></h1>
+<h4 align="center">Clean and Consistent CLI for your Ubuntu Advantage Systems</h4>
```

Comment:
Something important to review here is that I added a tagline. Is this acceptable? @renanrodrigo @lucasmoura

Reply:
Sure, I think it is ok

Reply:
+1